.ado/publish.yml

```diff
@@ -249,9 +249,8 @@ jobs:
         inputs:
           alertWarningLevel: Medium
           scanType: 'Register'
-          # Uncomment this once we have codesigning working
-          # ${{ if endsWith(variables['Build.SourceBranchName'], '-stable') }}: # Fail on stable release builds
-          #   failOnAlert: true
+          ${{ if endsWith(variables['Build.SourceBranchName'], '-stable') }}: # Fail on stable release builds
```

Comment:
nit: We should probably have a global variable "enableSigning" that is intiialzied with this expression to make it easier to reason about and test/disable.